frontend/src/metabase/archive/components/ArchivedEntityBanner/ArchivedEntityBanner.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import { CollectionPickerModal } from "metabase/common/components/CollectionPicker";
+import { Button, Flex, Icon, Paper, Text } from "metabase/ui";
+import type { CollectionId } from "metabase-types/api";
+
+type ArchivedEntityBannerProps = {
+  name: string;
+  entityType: string;
+  canWrite: boolean;
+  canRestore: boolean;
+  onUnarchive: () => void;
+  onMove: (id: CollectionId) => void;
+  onDeletePermanently: () => void;
+};
+
+export const ArchivedEntityBanner = ({
+  name,
+  entityType,
+  canRestore,
+  canWrite,
+  onUnarchive,
+  onMove,
+  onDeletePermanently,
+}: ArchivedEntityBannerProps) => {
+  const [showMovemodal, setShowMoveModal] = useState(false);
+
+  return (
+    <>
+      <Paper
+        px="1.5rem"
+        py=".75rem"
+        bg="error"
+        shadow="0"
+        radius="0"
+        w="100%"
+        data-testid="archive-banner"
+      >
+        <Flex justify="space-between">
+          <Flex align="center">
+            <Icon
+              color="white"
+              name="trash_filled"
+              style={{ marginInlineEnd: "1rem" }}
+            />
+            <Text color="white" size="md" lh="1rem">
+              {t`This ${entityType} is in the trash. `}
```

Comment:
Would it be easy to do:

```
{"dashboard": t`This dashboard is in the trash.`, "collection": t`This collection is in the trash.`, etc.}[entityType]
```

Then the translations will be accurate

Reply:
```
c("{0} is the entity in the trash, e.g. collection, dashboard, etc.").t`This ${entityType} is in the trash.`
```
How does this sound? I think we want to expand this to snippets, actions, etc. in the future. I'd prefer to keep the code where I don't have to come back and change it.

Reply:
My general view is that we should support i18n to a "good enough" standard, but you as the PR author basically get to decide what "good enough" means. So this is totally up to you.

My two cents, which you are free to take or leave: I think it's not too much of a headache to provide excellent support to the translators here, and provide a fallback for new entityTypes added later. That way, if you remember to come back and spell out the msgid for "snippet", "action", etc., great, but in the meantime those can use "this {0} is in the trash". 

Why I care: A sentence with a grammatical gender mismatch, like "Ce collection est à la poubelle," sounds pretty dumb, like "This collection are in the trash". I think we should avoid generating it if we can do so without twisting the code into terrible knots